ui/src/app/modals/os-welcome/os-welcome.page.html

```diff
@@ -6,27 +6,29 @@
       </ion-button>
     </ion-buttons>
     <ion-title >
-      <ion-label style="font-size: 20px;" class="ion-text-wrap">Welcome to {{ version }}</ion-label>
+      <ion-label style="font-size: 20px;" class="ion-text-wrap">Welcome to EmbassyOS {{ version }}!</ion-label>
     </ion-title>
   </ion-toolbar>
 </ion-header>
 
 <ion-content class="ion-padding">
-  <div>
-    <h3>Overview</h3>
-    <p>
-      0.2.8 is a small but important update designed to enhance awareness around potential pitfalls of using certain services.
-      It introduces warnings for installing, uninstalling, backing up, and restoring backups of stateful services such as LND or c-lightning.
-      This release also draws a distinction between services that can be launched inside the browser and those that are designed to run in the background.
-    </p>
-  </div>
+  <h2>Highlights</h2>
+  <p>
+    0.2.8 is a small but important update designed to enhance awareness around potential pitfalls of using certain services.
+    It introduces warnings for installing, uninstalling, backing up, and restoring backups of stateful services such as LND or c-lightning.
+    0.2.8 introduces automatic checks for updates, a setting that can be enabled or disabled in your Embassy config, and it also draws a distinction between services that are designed to be launched inside the browser and those that are designed to run in the background.
```

Comment:
You're not giving UI launching the attention it deserves in these notes. For most users, it will be the only change they notice.